Library/Formula/fig.rb

```diff
@@ -56,12 +54,9 @@ def install
       r.stage { system "python", "setup.py", "install", "--prefix=#{libexec}" }
     end
 
-    system "python", "setup.py", "install", "--prefix=#{prefix}"
-
-    rm "#{lib}/python2.7/site-packages/site.py"
-    rm "#{lib}/python2.7/site-packages/easy-install.pth"
+    system "python", "setup.py", "install", "--prefix=#{libexec}"
 
-    bin.env_script_all_files(libexec+"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+    (bin/"fig").write_env_script libexec/"bin/fig", :PYTHONPATH => ENV["PYTHONPATH"]
```

Comment:
If a keep this line it will install `wsdump.py` from package `websocket-client` and will conflict with tutum.